Content.Server/_NF/Holiday/HalloweenSystem.cs

```diff
@@ -0,0 +1,39 @@
+using System.Linq;
+using Robust.Shared.Map.Events;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Holiday;
+
+public sealed class HalloweenSystem : EntitySystem
```

Comment:
this is basically HolidaySystem, we dont need a new system for halloween only

Reply:
It is not basically HolidaySystem, no.